### PR TITLE
New package: WacomTechnology,Corp.WacomTabletDriver version 6.4.4-3

### PR DESCRIPTION
**Pull request by matthew-rutigliano-wacom**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/125746)

### CODE REVIEW

**Comment by matthew-rutigliano-wacom** [comment]
@microsoft-github-policy-service agree [company="Wacom Technology, Corp."]

**Comment by matthew-rutigliano-wacom** [comment]
@microsoft-github-policy-service agree company="Wacom Technology, Corp."

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 1:16:54 PM:  > 2023-11-14 13:16:53.780 [CLI ] Package hash verification failed. SHA256 in manifest [ba30d4eee5af309f0019d432f4a03b6057c0e07021de39fce9695c3a8c182683] does not match download [6e32f37bb0c6b8ca2787a7df225f17011bfa56469065a2eaf584419666951cb4]

**Comment by stephengillie** [comment]
It appears the hash has updated:
```PowerShell
PS C:\ManVal> Get-FileHash .\Installers\WacomTablet_6.4.4-3.exe

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          6E32F37BB0C6B8CA2787A7DF225F17011BFA56469065A2EAF584419666951CB4       C:\ManVal\Installers\WacomTable…
```

File: manifests/w/WacomTechnology,Corp/WacomTabletDriver/6.4.4-3/WacomTechnology,Corp.WacomTabletDriver.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WacomTechnology,Corp.WacomTabletDriver
+PackageVersion: 6.4.4-3
+InstallerType: exe
+Installers:
+- InstallerUrl: http://www.wacom.com/services/wacom/get-download-url.aspx?plat=win&dver=6.3.20-2&dt=drivers&redirect=true
+  Architecture: x64
+  InstallerSha256: BA30D4EEE5AF309F0019D432F4A03B6057C0E07021DE39FCE9695C3A8C182683
```
Comment (stephengillie) [inline manifests/w/WacomTechnology,Corp/WacomTabletDriver/6.4.4-3/WacomTechnology,Corp.WacomTabletDriver.installer.yaml:10]:
```suggestion
  InstallerSha256: 6E32F37BB0C6B8CA2787A7DF225F17011BFA56469065A2EAF584419666951CB4
```

Comment (stephengillie) [inline manifests/w/WacomTechnology,Corp/WacomTabletDriver/6.4.4-3/WacomTechnology,Corp.WacomTabletDriver.installer.yaml:10]:
```suggestion
  InstallerSha256: 6E32F37BB0C6B8CA2787A7DF225F17011BFA56469065A2EAF584419666951CB4
```

File: manifests/w/WacomTechnology,Corp/WacomTabletDriver/6.4.4-3/WacomTechnology,Corp.WacomTabletDriver.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WacomTechnology,Corp.WacomTabletDriver
+PackageVersion: 6.4.4-3
+InstallerType: exe
+Installers:
+- InstallerUrl: http://www.wacom.com/services/wacom/get-download-url.aspx?plat=win&dver=6.3.20-2&dt=drivers&redirect=true
```
Comment (stephengillie) [inline manifests/w/WacomTechnology,Corp/WacomTabletDriver/6.4.4-3/WacomTechnology,Corp.WacomTabletDriver.installer.yaml:8]:
```suggestion
- InstallerUrl: http://www.wacom.com/services/wacom/get-download-url.aspx?plat=win&dver=6.4.4-3&dt=drivers&redirect=true
```
While the URL with `6.3.20-2` retrieves the `6.4.4-3` version, would it be better to have the version in the URL?

**Pull request closed without merging**